stories/topics/proc-aap-aws-backup-and-recovery.adoc

```diff
@@ -0,0 +1,68 @@
+
+=== Take a manual snapshot of the RDS for your AAP deployment
+. Navigate to *CloudFormation* on your AWS Console
+. Select the *Resources* tab and navigate to menu:rds[rds] on the resource tree and click the link to go to the RDS of the deployment
+. Select *Instance actions* near the top of the screen to reveal a drop-down menu and click the btn:[Take Snapshot] button
+. Enter a name for the snapshot and click btn:[Take Snapshot]
+. Your snapshot should now appear in a list of manual RDS snapshots. It is important to take a manual snapshot over relying on the automated ones created by an instance as automated snapshots are attached to the lifecycle of the RDS instance. An automated snapshot will not be available to be used for recovery in the event your deployment is lost
+
+== Recovery Process
+
+=== Find ARN of RDS snapshot you want to recover from
+. Navigate to *RDS* on your AWS console
+. Select *Snapshots* in the left menu
+. Select the snapshot you want to use for your new deployment
+. The ARN will be listed near the top right of the page. Record the ARN of the snapshot for use in the next step
+
+=== Modify existing CloudFormation Template to reference an RDS Snapshot
+. Find the CloudFormation Template used to create your old AAP Deployment
+. Navigate to the RDS component of the CloudFormation Template (Can search for *postgres* to get to the component)
+. Under `Properties`` add `“DBSnapshotIdentifier”: <ARN of the RDS snapshot>`
+. Under `Properties`` change `“DBName”: “awx”` to  `“DBName”: {“Ref": "AWS::NoValue" }`
+
+=== Restore an EFS Backup for your deployment
+. In the AWS Console, navigate to *CloudFormation* and select your deployment
+. Select the *Resources* tab and expand the *storage* section on the resource tree
+. Record the physical ID of the filesystem (format should look like this fs-044fce19b58a6d8f5)
+. Navigate to *AWS Backup* on your AWS Console
+. On the side bar select *Backup vaults*
+. From the displayed list of backup vaults select *aws/efs/automatic-backup-vault*
+. Filter recovery points by entering the filesystem id, found in the step above, in to the filter bar
+. Select a recovery point from the filtered results. (Recovery points are sorted from most to least recent)
+. Click the btn:[Restore] Button in the top right corner of the page
+. For *Restore type* select *Full restore*
+. For *Restore location* select *Restore to a new file system*
+. For *File system type* select *Regional*
+. For *Performance* select *General Purpose*
+. Select the *Enable Encryption* radio button
+. Wait for the Restore Job to complete and record the Resource ID of the restore job. The format should look like “file-system/fs-08729f158d58b3c77”. We want to record everything to the right of the slash, so in this case we would record fs-08729f158d58b3c77. This is the id of the restored EFS filesystem and it will be important for the next step.
+
+=== Modify existing CloudFormation Template to reference restored EFS
```

Comment:
To modify an existing CloudFormation Template to reference a restored EFS.

.Procedure

Reply:
"Modify the CloudFormation Template of your current {PlatformNameShort} Deployment to reference an RDS Snapshot"

I think this might be a better option than using "To modify" because I feel like "To modify" makes the step seem optional and not a part of the entire recovery procedure. In order for the customer to successfully restore their deployment they must complete every step.

Reply:
I see. OK.